server/src/domain/metadata/metadata.service.ts

```diff
@@ -431,7 +432,18 @@ export class MetadataService {
   ): Promise<{ exifData: ExifEntityWithoutGeocodeAndTypeOrm; tags: ImmichTags }> {
     const stats = await this.storageRepository.stat(asset.originalPath);
     const mediaTags = await this.repository.readTags(asset.originalPath);
-    const sidecarTags = asset.sidecarPath ? await this.repository.readTags(asset.sidecarPath) : null;
+
+    let sidecarTags = null;
+    if (asset.sidecarPath) {
+      // check if sidecar-file exists
+      if (fs.existsSync(asset.sidecarPath)) {
+        sidecarTags = await this.repository.readTags(asset.sidecarPath);
+      } else {
+        this.logger.error(
+          `Sidecar-File '${asset.sidecarPath}' for asset '${asset.id}' does not exist on the filesystem`,
```

Comment:
Is an error/warning enough or does the db-field need to be nulled when not found?